modules/vye/app/models/vye/user_info.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Vye; end
+
+class Vye::UserInfo < ApplicationRecord
+  class DobSerializer
+    def self.load(v)
+      Date.parse(v) if v.present?
+    end
+
+    def self.dump(v)
+      v.to_s if v.present?
+    end
+  end
+
+  include Vye::GenDigest
+
+  has_many :address_changes
+  has_many :awards
+  has_many :direct_deposit_changes
+  has_many :pending_documents, foreign_key: :ssn_digest, primary_key: :ssn_digest, inverse_of: :user_info
+  has_many :verifications
+
+  ENCRYPTED_ATTRIBUTES = %i[
+    address_line2 address_line3 address_line4 address_line5 address_line6 dob file_number full_name ssn stub_nm zip
```

Comment:
Sorry, I can't quite remember, but I thought SSN wasn't going to be directly stored?

Reply:
SSNs are hashed, and encrypted